app/workers/education_form/process_10203_submissions.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+require 'evss/gi_bill_status/service'
+require 'sentry_logging'
+
+module EducationForm
+  DENIED = 'denied'
+  PROCESSED = 'processed'
+  INIT = 'init'
+
+  class Process10203SubmissionsLogging < StandardError
+  end
+
+  class Process10203Submissions
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options queue: 'default',
+                    unique_for: 30.minutes,
+                    retry: 5,
+                    backtrace: true
+
+    # Get all 10203 submissions that have a row in education_stem_automated_decisions
+    def perform(
+      records: EducationBenefitsClaim.joins(:education_stem_automated_decision).includes(:saved_claim).where(
+        saved_claims: {
+          form_id: '22-10203'
+        }
+      )
+    )
+      return unless Flipper.enabled?(:stem_automated_decision)
+
+      if records.count.zero?
+        log_info('No records to process.')
+        return true
+      else
+        log_info("Processing #{records.count} application(s)")
+      end
+
+      user_submissions = group_user_uuid(records)
+      process_user_submissions(user_submissions)
+    end
+
+    private
+
+    # Group the submissions by user_uuid
+    def group_user_uuid(records)
+      records.group_by { |ebc| ebc.education_stem_automated_decision&.user_uuid }
+    end
+
+    # If the user doesn't have EVSS data mark the 10203 as DENIED
+    # If there are multiple submissions for a user compare un-submitted to most recent processed
+    #   by EducationForm::CreateDailySpoolFiles
+    # Otherwise check submission data and EVSS data to see if submission can be marked as PROCESSED
+    def process_user_submissions(user_submissions)
+      user_submissions.each do |user_uuid, submissions|
+        gi_bill_status = get_gi_bill_status(user_uuid)
+        if gi_bill_status == {} || gi_bill_status.remaining_entitlement.blank?
+          submissions.each { |submission| update_status(submission, DENIED) }
```

Comment:
We cannot auto deny based on no EVSS. These should be processed as 10203.

Reply:
fixed